packages/twenty-front/package.json

```diff
@@ -59,5 +59,8 @@
   },
   "msw": {
     "workerDirectory": "public"
+  },
+  "devDependencies": {
+    "storybook-addon-react-router-v6": "^2.0.10"
```

Comment:
provide an efficient way to mock react router in storybook tests

Reply:
removed it and used MemoryRouter instead since it does not add a dependency + it is used in other places